plugins/system/pressure

```diff
@@ -0,0 +1,331 @@
+    printf "multigraph pressure.%s_total\n" $resource
+    printf "graph_title %s Pressure Stall Information - Rate\n" $printable_resource
+    printf "graph_category system\n"
+    printf "graph_info Total PSI based latency rate caused by lack of %s resources.\n" $printable_resource
+    printf "graph_vlabel rate\n"
+    interval="total"
+    output_config $resource $interval
+    echo ""
+  done
+  
+  printf "multigraph pressure\n"
+  printf "graph_title Pressure Stall Information - Average\n"
+  printf "graph_vlabel %%\n"
+  printf "graph_scale no\n"
+  printf "graph_category system\n"
+  printf "graph_info Average PSI based latency caused by lack of resources.\n"
+  for resource in ${pressure_resources[@]}; do
+    output_config $resource $summary_interval
+  done
+  echo ""
+}
+
+iterate_values() {
+  for resource in ${pressure_resources[@]}; do
+    printf "multigraph pressure.%s_avg\n" $resource
+    for interval in ${pressure_intervals[@]}; do
+      output_values $resource $interval
+    done
+    echo ""
+  done
+  
+  for resource in ${pressure_resources[@]}; do
+    printf "multigraph pressure.%s_total\n" $resource
+    interval="total"
+    output_values $resource $interval
+    echo ""
+  done
+  
+  printf "multigraph pressure\n"
+  for resource in ${pressure_resources[@]}; do
+    output_values $resource $summary_interval
+  done
+  echo ""
+}
+
+output_config() {
+  resource=$1
+  interval=$2
+  
+  printable_resource=$( get_printable_name resource $resource )
+  printable_interval=$( get_printable_name interval $interval )
+  
+  for scope in ${pressure_scopes[@]}; do
+    if [ ${resource} == "cpu" ] && [ ${scope} != "some" ]; then
+      continue
+    else
+      printable_scope=$( get_printable_name scope $scope )
+      printf "psi_%s_%s_%s.min 0\n" $resource $interval $scope
+      printf "psi_%s_%s_%s.label %s %s %s\n" $resource $interval $scope $printable_resource $printable_interval $printable_scope
+      this_warn_var=$( echo "warn_psi_${resource}_${interval}_${scope}" | sed 's/[^A-Za-z0-9_]/_/g' )
+      if [ -n "${!this_warn_var}" ]; then
+        printf "psi_%s_%s_%s.warning %s\n" $resource $interval $scope ${!this_warn_var}
+      fi
+      this_crit_var=$( echo "crit_psi_${resource}_${interval}_${scope}" | sed 's/[^A-Za-z0-9_]/_/g' )
+      if [ -n "${!this_crit_var}" ]; then
+        printf "psi_%s_%s_%s.critical %s\n" $resource $interval $scope ${!this_crit_var}
+      fi
+      if [ $interval == "total" ]; then
+        printf "psi_%s_%s_%s.type DERIVE\n" $resource $interval $scope
+      fi
+    fi
+  done
+}
+
+output_values() {
+  resource=$1
+  interval=$2
+  
+  for scope in ${pressure_scopes[@]}; do
+    if [ ${resource} == "cpu" ] && [ ${scope} != "some" ]; then
+      continue
+    else
+      printf "psi_%s_%s_%s.value %s\n" $resource $interval $scope $(get_pressure_value $resource $interval $scope)
+    fi
+  done
+}
+
+output_usage() {
```

Comment:
Maybe simply redirect `output_usage` to *stderr* when it is called?
(in order to remove all these redirects here)